core/embed/rust/src/ui/model_tt/component/error.rs

```diff
@@ -0,0 +1,104 @@
+use crate::ui::{
+    component::{
+        text::paragraphs::{ParagraphStrType, ParagraphVecShort, Paragraphs},
+        Child, Component, ComponentExt, Event, EventCtx, Label, Never, Pad,
+    },
+    constant::screen,
+    display::{self, Icon},
+    geometry::{Alignment::Center, Point, Rect, TOP_CENTER},
+};
+
+use crate::ui::model_tt::{
+    constant::WIDTH,
+    theme::{
+        FATAL_ERROR_COLOR, FATAL_ERROR_HIGHLIGHT_COLOR, ICON_WARNING40, TEXT_ERROR_BOLD, WHITE,
+    },
+};
+
+const PADDING: i16 = 6;
+const FOOTER_START: i16 = 178;
+const FOOTER_HEIGHT: i16 = 56;
+
+pub struct ErrorScreen<T> {
+    bg: Pad,
+    small_pad: Pad,
```

Comment:
Could this be called `footer_pad`? It is being placed on the same area

Reply:
removed, see below.

https://github.com/trezor/trezor-firmware/commit/4acf54ce632927f4117f1491055323fbbc73c50b

in result.rs i renamed it: https://github.com/trezor/trezor-firmware/commit/7397275c9d65c0946e0c7c6fc70a0f88194d3282